src/routes/news/components/NewsElement.js

```diff
@@ -74,7 +74,7 @@ class NewsElement extends React.PureComponent<PropsType> {
 
   render () {
     const { path, id } = this.props
-    const itemPath = `${path}/${id}`
+    const itemPath = `${path}${id}`
```

Comment:
Path already contains a slash or why this change?

Reply:
Yep. It wasn't working with the slash.

Reply:
What exactly was not working? It's working for me (to click on a news element in the news list).

Reply:
@markl told me he already fixed this somewhere else. I'll remove it.